examples/exoskeleton/js/views/app-view.js

```diff
@@ -8,6 +8,12 @@ var app = app || {};
 		el.style.display = toggle ? '' : 'none';
 	};
 
+	var matchesSelector = function (node, selector) {
+		return [].some.call(document.querySelectorAll(selector), function (el) {
```

Comment:
`Array.prototype.some` then no need to allocate a new array instance just to use the method eh :)